torch_xla/csrc/data_ops.h

```diff
@@ -25,6 +25,10 @@ std::vector<int64_t> GetCompleteShape(absl::Span<const int64_t> output_sizes,
 // output size.
 xla::XlaOp BuildView(xla::XlaOp input, absl::Span<const int64_t> output_sizes);
 
+// Update Output Dynamic Dimensions based on input size()
```

Comment:
this comment is somewhat confusing, the parameter is called `output_sizes` and the comment refer to `input size()`. Maybe just something along the line of `Return a new XlaOp that reflects pass in size`?